docs/src/main/sphinx/routines/introduction.md

```diff
@@ -66,6 +66,8 @@ In this scenario, the following commands can be used:
 * [](/sql/create-function) to create and store a routine.
 * [](/sql/drop-function) to remove a routine.
 * [](/sql/show-functions) to display a list of routines in a catalog.
+* [](/sql/show-create-function) to show the source code of the routine that
```

Comment:
to display the definition of a SQL catalog routine.

Confirm that it only works for catalog routines... btw